### PR TITLE
Remove additional environment requirements from Cloudnative e2e test

### PR DESCRIPTION
**Pull request by aorcholski**
# Description

Tests requiring additional installs on the cluster are removed from the default CloudNative test. There are 4 CN tests:
- cloudnative (`make test/e2e/cloudnative`) - the default one, no additional requirements
- cloudnative/istio  (`make test/e2e/cloudnative/istio`) , istio required
- cloudnative/network (`make test/e2e/cloudnative/network`) , cilium required
- cloudnative/proxy (`make test/e2e/cloudnative/proxy`) , cilium required

`testdata/*` paths are relative to absolute `project root directory` now. The Project Root Directory is detected in the runtime during module initialization phase.

## How can this be tested?
Use `make test/e2e/cloudnative` or `make test/e2e/cloudnative/<name>` commands to start a test. Make sure you have  properly configured cluster.

## Checklist
- [x] PR is labeled accordingly
- [x] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by 0sewa0 — COMMENTED** [review]
we should up the default timeout from 10m to 20m of the default `test/e2e/cloudnative` test case
as it has been growing, while the timeout remained the same.

**Comment by waodim** [comment]
I saw that the file that holds is called `cloudnative_proxy_test.go` - do we want to keep it like that or also call it `coudnative_test.go` like the others?

**Review by 0sewa0 — APPROVED** [review]

File: test/project/project.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	"path"
+)
+
+var rootDir string
+
+func RootDir() string {
+	return rootDir
+}
+
+func TestDataDir() string {
```
Comment (chrismuellner) [inline test/project/project.go:14]: would https://pkg.go.dev/os#Executable point to the right location?

Comment (chrismuellner) [reply test/project/project.go:14]: thanks for making sure 🙂

**Pull request merged**